public/docs/ts/latest/guide/learning-angular.jade

```diff
@@ -0,0 +1,35 @@
+block includes
+  include ../_util-fns
+
+- var langName = current.path[1] == 'ts' ? 'TypeScript' : 'JavaScript'
+figure
+  img(src="/resources/images/devguide/intro/people.png" alt="Us" align="left" style="width:200px; margin-left:-40px;margin-right:10px" )
+:marked
+  Everyone is different. Everyone learns differently.
+  
+  You don't have to read the documentation straight through.  Most pages stand on their own.
+  Those new to Angular may wish to follow this popular learning path.
+  <br clear="all">

```

Comment:
This should be `<br class="l-clear-left">` (since `clear="all"` is deprecated).
Also, please put this `<br...>` at the end of the previous line (one of the jade translators needs that trigger).